### PR TITLE
Minor improvements

### PR DESCRIPTION
**Pull request by santib**

### CODE REVIEW

**Review by mrodriguez90 — APPROVED** [review]
🙌

**Review by JulianPasquale — APPROVED** [review]

File: Gemfile [hunk 1]
```diff
@@ -69,7 +70,6 @@ end
 
 group :test do
   gem 'capybara', '~> 3.39', '>= 3.39.2'
-  gem 'faker', '~> 3.2'
```
Comment (blacksam07) [inline Gemfile:72]: We remove `Faker` gem but not update the existing factories, we want to remove it ?

**Review by blacksam07 — CHANGES_REQUESTED** [review]

**Review by PerezIgnacio — APPROVED** [review]

File: Gemfile (hunk 1 above)
Comment (PerezIgnacio) [reply Gemfile:72]: It was moved to dev, test group

Comment (blacksam07) [reply Gemfile:72]: I didn't see it

**Review by blacksam07 — APPROVED** [review]
👏🏽

**Pull request merged**